libs/algorithms/CMakeLists.txt

```diff
@@ -144,3 +144,6 @@ add_hpx_module(algorithms
   CMAKE_SUBDIRS examples tests
 )
 
+include(HPX_PrintSummary)
+create_configuration_summary(
+  "  Module configuration summary (algorithms):" "algorithms")
```

Comment:
Can we move these into the `add_hpx_module` function?

Reply:
Sure, good thinking!